### ISSUE TITLE
Region of Interest at test time

### ISSUE THREAD

**Issue by IssamLaradji** [leeyeehoo/CSRNet-pytorch#14]
Great code thank you!

For datasets with regions of interest (ROI) such as UCSD and world Expo, do you use the ROI to mask the input image at test time? I think it makes sense that the ROI is accessible at test time (as it is unique to each scene/camera). Thanks.

**Comment by leeyeehoo** [comment]
Yes. We first put ROI masks on the training dataset. And test with ROI. Thanks.

**Comment by IssamLaradji** [comment]
Thanks!
